packages/design-system/src/components/contextMenu/components/subMenuTrigger.tsx

```diff
@@ -100,20 +112,20 @@ function SubMenuTrigger({
     if (!isSubMenuOpen || !node) {
       return undefined;
     }
-    const onPointerMove = (e) => {
-      // Track the pointer when moving inside the menu while the submenu is open.
-      pointerTracker.current.x = e.clientX;
-      pointerTracker.current.y = e.clientY;
-      maybeCloseSubMenu();
+    const onPointerMove = (e: Event) => {
```

Comment:
Should it just be `PointerEvent` directly perhaps?

Reply:
This is a native HTML event listener - it is only typed as dispatching a generic event, not typed specifically for each individual event type. Putting `PointerEvent` would result in this error a few lines below.

```
Argument of type '(e: PointerEvent) => void' is not assignable to parameter of type 'EventListenerOrEventListenerObject'.